packages/ns-api/files/ns.snort

```diff
@@ -464,13 +488,18 @@ if cmd == 'list':
         "suppress-alert": {"gid": 1, "sid": 100000, "direction": "by_src", "ip": "*.*.*.*", "description": "Description"},
         "delete-suppression": {"gid": 1, "sid": 100000, "direction": "by_src", "ip": "*.*.*.*"},
         "list-events": {},
+        "status": {
+            "enabled": True,
+            "alerts": 10,
+            "drops": 10
+        }
```

Comment:
Status does not get any input.
Remember also to update the README with recent changes